server/src/main/java/org/apache/druid/server/coordinator/loading/LoadQueueTaskMaster.java

```diff
@@ -45,6 +53,8 @@ public class LoadQueueTaskMaster
   private final ZkPathsConfig zkPaths;
   private final boolean httpLoading;
 
+  private final ConcurrentHashMap<String, LoadQueuePeon> loadManagementPeons = new ConcurrentHashMap<>();
```

Comment:
If stopAndRemoveAllPeons is called by the Coordinator when it is no longer the leader and the `PrepareBalancerAndLoadQueues` duty which was scheduled previously is running simultaneously, could there be problems?
(I think not since the coordinator would not run any further duties)